input/fsh/au-erequesting-diagnosticrequest.fsh

```diff
@@ -1,7 +1,7 @@
-Profile: AUeRequestingServiceRequest
+Profile: AUeRequestingDiagnosticRequest
 Parent: AUBaseDiagnosticRequest
-Id: au-erequesting-servicerequest
-Title: "AU eRequesting ServiceRequest"
```

Comment:
Commenting here to get an inline comment. 
Please update theAUeREqDI mappings into AUeRequesting table: https://build.fhir.org/ig/hl7au/au-fhir-erequesting/branches/master/auereqdi.html#auereqdi-mappings-into-au-erequesting

Reply:
Amended, thanks.